src/ase/cw/gui/ServerFrame.java

```diff
@@ -12,20 +16,22 @@
 public class ServerFrame extends JFrame implements ActionListener, ServerView {
 
     private JPanel content = new JPanel(new BorderLayout(10, 10));
-    private JTextArea textArea = new JTextArea("Status: free");
+    private JTextArea textArea = new JTextArea("");
 
     // Frame constructor
-    public ServerFrame(int server_id) {
+    public ServerFrame(Server server, JFrame parentFrame) {
```

Comment:
Why is Server a paremeter here?
We do not need a Server parameter.
You only need server for a Title and a short Log message?

So why do you not use 
public ServerFrame(String title0, JFrame parentFrame) ?

Furthermore, the ID in server is not needed at all.